features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/ServiceCatalog/ServiceCatalogComponent.jsx

```diff
@@ -18,22 +18,29 @@
 
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
-import Listing from './Listing/Listing';
+import Listing from 'AppComponents/ServiceCatalog/Listing/Listing';
+import Overview from 'AppComponents/ServiceCatalog/Listing/Overview';
 
 /**
  * Have used key={Date.now()} for `Route` element in `/service-catalog`
  */
-const ServiceCatalog = () => {
+const ServiceCatalogComponent = () => {
     return (
         <Switch>
             <Route
                 exact
                 path='/service-catalog'
                 key={Date.now()}
-                component={Listing}
+                component={(props) => <Listing {...props} />}
+            />
+            <Route
+                path='/service-catalog/:service_uuid/overview'
+                key='/service-catalog/:service_uuid/overview'
+                // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
Please don't disable the prop spread rule, Pass only the required props,

And in here, we don't need to use a Lambda component here, Just put thee component there and that's it